tests/test_trigger_batch_interval/results_true.dat

```diff
@@ -1,27 +1,28 @@
 k-combined:
 9.851940E-01 4.283951E-03
-tallies:
+tally 1:
```

Comment:
All the same numbers are there, they've just been interleaved in a different order.